### PR TITLE
fix: allow Fn::ToBase64 relax parsing rule, allow arbitrary expressions that eval to string

### PR DESCRIPTION
**Pull request by AaronFriel**

### CODE REVIEW

**Comment by AaronFriel** [comment]
Ah, this was a sloppy PR, my mistake. Fixing up the syntax & running tests.

**Review by dixler — APPROVED** [review]
LGTM

File: CHANGELOG_PENDING.md [hunk 1]
```diff
@@ -13,5 +13,9 @@
 - De-duplicate error message added during pre-eval checking.  
   [#207](https://github.com/pulumi/pulumi-yaml/pull/207)
 
-- Prevent invokes without inputs from crashing `pulumi-language-yaml`.  
+- Prevent invokes without inputs from crashing `pulumi-language-yaml`.
+  [#216](https://github.com/pulumi/pulumi-yaml/pull/216)
+
+- Allow Fn::ToBase64 to take expressions as inputs, was previously constrained to only allow a
+  string constant.
   [#216](https://github.com/pulumi/pulumi-yaml/pull/216)
```
Comment (iwahbe) [inline CHANGELOG_PENDING.md:21]: This looks like the wrong linked PR

Comment (AaronFriel) [inline CHANGELOG_PENDING.md:21]:
```suggestion
  [#221](https://github.com/pulumi/pulumi-yaml/pull/221)
```

Comment (AaronFriel) [reply CHANGELOG_PENDING.md]: fixed!

**Pull request merged**